apps/design/backend/package.json

```diff
@@ -21,7 +19,12 @@
     "clean:self": "rm -rf build && tsc --build --clean tsconfig.build.json",
     "create-org": "node -r esbuild-runner/register ./scripts/create_org.ts",
     "create-user": "node -r esbuild-runner/register ./scripts/create_user.ts",
-    "db:reset": "./scripts/db_reset_dev.sh",
+    "db:migrations:create": "node-pg-migrate create",
+    "db:migrations:run": "node-pg-migrate up",
+    "db:migrations:run-dev": "DATABASE_URL='postgresql://design:design@localhost:5432/design' node-pg-migrate up",
+    "db:migrations:undo-last": "node-pg-migrate down",
```

Comment:
How do the down migrations work? I see that node-pg-migrate claims it can infer them, but I imagine there are a bunch of cases where it's not obvious without writing a reverse migration how to undo.

In Arlo we just decided to forgo down migrations and have never needed them. I don't think it's something we need to decide right now, but mostly just wondering your thoughts